src/views/order_list_view.js

```diff
@@ -0,0 +1,69 @@
+import Backbone from 'backbone';
+import _ from 'underscore';
+
+import Order from '../models/order'
+import OrderView from './order_view';
+
+
+const OrderListView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+
+    this.listenTo(this.model, 'update', this.render)
+
+    this.listenTo(this.bus, 'newOrder', this.addOrder);
+    this.listenTo(this.bus, 'newOrder', this.render);
```

Comment:
In general it's totally okay to have multiple callbacks for a single event (this is especially useful when the callbacks are doing completely unrelated stuff), however in this case it's not necessary.

What will happen hear is that `this.addOrder()` will be called first when the `newOrder` event is triggered. When that method runs it calls `this.model.add()`, which puts the new order into the `OrderList` collection... which itself triggers the `update` event on the collection.

And we already have a listener for the `update` event which runs `this.render()`, so we're effectively calling `this.render()` twice. Once as a result of `update` and then again as a result of `newOrder`.